client/app/bundles/course/assessment/submission/actions/index.js

```diff
@@ -16,6 +16,29 @@ export function setNotification(message, errors) {
   };
 }
 
+const formatAnswer = (answer) => {
```

Comment:
can add some documentation to this method? i.e. why we need to format

Reply:
Sure, will add the documentation.